deploy/kubernetes/seqr/seqr.gcloud.yaml

```diff
@@ -154,14 +154,8 @@ spec:
           value: "{{ ELASTICSEARCH_SERVICE_PORT }}"
         - name: KIBANA_SERVICE_PORT
           value: "{{ KIBANA_SERVICE_PORT }}"
-        - name: DEPLOY_TO
-          value: "{{ DEPLOY_TO }}"
-        - name: DEPLOY_TO_PREFIX
-          value: "{{ DEPLOY_TO_PREFIX }}"
         - name: DEPLOYMENT_TYPE
           value: "{{ DEPLOYMENT_TYPE }}"
-        - name: CLUSTER_NAME
-          value: "{{ CLUSTER_NAME }}"
```

Comment:
I'm not sure the reason for delete `DEPLOY_TO` and `CLUSTER_NAME`.

Reply:
my understanding here was that the seqr pod itself doesn't actually need to know those values in the environment at runtime. but I could be wrong...

Reply:
they are never used